server/index.ts

```diff
@@ -106,14 +106,12 @@ export async function startServer() {
 
   await Promise.all([app.prepare(), processUploads(), processDeployments()])
 
-  server.use((req, res) => {
-    return handle(req, res)
-  })
+  server.use((req, res) => handle(req, res))
 
   http.createServer(server).listen(port)
   logger.info({ port }, `Listening on port ${port}`)
 }
 
-if (require.main === module) {
+if (import.meta.url.startsWith('file:') && import.meta.url === pathToFileURL(process.argv[1]).href) {
```

Comment:
Oh my...

Reply:
Yep it's horrible isn't it. Feel free to suggest a better alternative if you can find one.

From the Node docs: `When the entry point is not a CommonJS module, require.main is undefined, and the main module is out of reach.` - https://nodejs.org/api/modules.html#accessing-the-main-module